addons/mass_mailing/models/mailing.py

```diff
@@ -148,7 +148,7 @@ def _get_default_mail_server_id(self):
     ab_testing_pc = fields.Integer(string='A/B Testing percentage',
         help='Percentage of the contacts that will be mailed. Recipients will be chosen randomly.', default=10)
     ab_testing_schedule_datetime = fields.Datetime(related="campaign_id.ab_testing_schedule_datetime", readonly=False,
-        default=lambda self: fields.Datetime.now() + relativedelta(days=1))
+        default=lambda self: fields.Datetime.now() + relativedelta(days=1), copy=True)
```

Comment:
Would maybe do it directly in 'action_duplicate' in which we would copy 'ab_testing_schedule_datetime' if 'ab_testing_enabled' is activated on source mailing to avoid duplicating unwanted information in other cases, unless you have strong reasons to do it at model level ?